From 1df147d5337622ea979eb1319c97d8278b70d864 Mon Sep 17 00:00:00 2001
From: Ralf Nolden <nolden@kde.org>
Date: Tue, 17 May 2016 12:47:59 +0200
Subject: [PATCH] Make qtmultimedia 5.6 branch compile without c++11:nullptr
 usage

5.6 should not require c++11 features, however, nullptr slipped in here.
Replace nullptr with Q_NULLPTR for initialization.

Change-Id: Icf62618657b5bb725bd4c8f924853fa191f6e413
Reviewed-by: Simon Hausmann <simon.hausmann@theqtcompany.com>

URL: https://codereview.qt-project.org/#/c/159342/
--- src/gsttools/qgstreamerbushelper.cpp
+++ src/gsttools/qgstreamerbushelper.cpp
@@ -52,7 +52,7 @@ public:
         m_tag(0),
         m_bus(bus),
         m_helper(parent),
-        m_intervalTimer(nullptr)
+        m_intervalTimer(Q_NULLPTR)
     {
         // glib event loop can be disabled either by env variable or QT_NO_GLIB define, so check the dispacher
         QAbstractEventDispatcher *dispatcher = QCoreApplication::eventDispatcher();
-- 
2.8.1
